HeRoBackEnd/Controllers/CandidateController.cs

```diff
@@ -1,10 +1,13 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using HeRoBackEnd.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Services.Services;
+using Data.Entities;
 
 namespace HeRoBackEnd.Controllers
 {
     public class CandidateController : Controller
     {
-        private candidateService; 
```

Comment:
To się kompiluje?

Reply:
jeszcze nie mam dostępu do serwisów więc podkreśla na czerwono, ale powinno da się skompilować

Reply:
Jak podkreśla to znaczy, że nie możesz zbudować projektu. Zrób tak żeby nie było errorów i żeby build przeszedł pomyślnie. Czyli usuń wszystko czego jeszcze nie ma - candidateService, recruitmentService itd